alcs-api/src/user/user.controller.ts

```diff
@@ -26,13 +26,6 @@ export class UserController {
     return this.userService.createUser(dto);
```

Comment:
Should we also delete create user since it should only happen from SSO now?

Reply:
that is a good question. We do not have any requirements on supporting something other than SSO so I think yes. I can get rid of it in my next pr

Reply:
Sounds good